canonical/memcached-operator
# DPE-78 Replaced charmhelpers.Template by Jinja2.Template

**What needs to be reviewed:**

1. charm.py _on_config_changed() - Replaces charmhelpers.Template by jinja2.Template
2. test_charm.py - Implements all unit tests for previous method

**How to test**
Functionally, this version should be identical to the previous, therefore the testing steps are the same:

1. `charmcraft pack` and `juju deploy` as usual
2. Use `juju status` to find out assigned IP address, the port is currently 11211
3. Use ``telnet <IP_address> 11211`
4. Once connected, type stats followed by ENTER
5. If you want to try other commands, see [this page](https://techleader.pro/a/90-Accessing-Memcached-from-the-command-line)
6. Once done, type quit followed by ENTER
